python/ray/llm/tests/conftest.py

```diff
@@ -136,7 +136,7 @@ def gpu_type():
         print(f"{torch.version.cuda=}", flush=True)
         name = torch.cuda.get_device_name()
         # The name of the GPU is in the format of "NVIDIA L4" or "Tesla T4".
-        _, type_name = name.split(" ")
+        type_name = name.split(" ")[1]
```

Comment:
Why change this line?

Reply:
Sometimes the `name` will be like "NVIDIA H100 80GB HBM3", then it will raise an error `Failed to get the GPU type: too many values to unpack (expected 2)`

Reply:
Ah I see. Can you comment this case to the code for reference?

Reply:
done

Reply:
Something like this `re.compile(r'(?:NVIDIA|Tesla)\s+(L4|T4|H100)')` ?